backend/layers/processing/schema_migration.py

```diff
@@ -224,7 +224,7 @@ def publish_and_cleanup(self, collection_version_id: str, can_publish: bool) ->
         if errors:
             self._store_sfn_response("report", collection_version_id, errors)
         elif can_publish:
-            self.business_logic.publish_collection_version(collection_version_id)
+            self.business_logic.publish_collection_version(collection_version)
```

Comment:
This value is set [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/d70ce570d359090e3b1aa84c94892ed06d1c68c8/backend/layers/processing/schema_migration.py#L185)